src/Raku/ast/code.rakumod

```diff
@@ -2684,6 +2686,10 @@ class RakuAST::CurryThunk
         '⋐' ~ nqp::x('🔆', self.IMPL-NUM-PARAMS)  ~ '⋑'
     }
 
+    method set-original-expression(Str $original-expression) {
```

Comment:
If the "original-expression" is mandatory with `.new`, we don't need this method, do we?  Or do we *change* the original expression at some point?

Reply:
You're correct, we actually do not need this setter.